resources/guides/laptop_guide.md

```diff
@@ -0,0 +1,72 @@
+
+If you are taking classes offered by other faculties or departments (e.g. Engineering), you may have to install software that are only available on Windows. However, this usually is not a concern since such instances are quite rare, and the University of Windsor has many other computer resources to help you out if something like this happens.
+
+### Linux
+
+Linux/UNIX is very good for programming/developing because of how easy it is to set up the necessary environment. Much like MacOS, it can be difficult to game on. Depending on how important gaming is to you, you may need to use Wine, set up virtual machine, or dual-boot with Windows. Additionally, since many of the University's software (Office 365, OneDrive, Teams, etc.) are on the Microsoft platform, there _may_ be some compatibility issues.
+
+Popular Linux distros include Ubuntu, Debian, Fedora, Pop!\_OS, Linux Mint, and Arch Linux. If you are very new to Linux, then Ubuntu and Debian may be a good starting point for you simply because there is a large community using them and most likely have answers for any problem you encounter. The University, for example, uses Debian on its remote servers. If you have time and are willing to experiment, you should test out the different Linux OS since the desktop environment on Ubuntu and Debian (GNOME) can be slightly laggy on laptops without high specifications.
+
+Although Linux OS may be confusing for new users, becoming comfortable with a UNIX/Linux environment is very beneficial for you. The University generally expects you to produce codes/projects that will work on its Linux server. As well, Linux forces you to interact with the command line more, which is great if you want to learn more about how your computer works underneath.
+
+Almost all Linux distros are free, especially the popular ones. As well, people have said that Linux takes up less storage and is more battery-efficient compared to Windows. Thus, the right Linux distro will significantly optimize your laptop, especially if you are on a budget.
+
+### MacOS
+
+Since the MacOS is a UNIX-based operating system, it is usually easier to set up local programming environment than Windows. As well, it is also among the most supported OS in terms of IT support and app availability. However, MacOS is generally not compatible with non-Apple products. This means that you would need to purchase a MacBook to use MacOS, which may be expensive/less cost-effective depending on your budget and what specifications you're looking for.
```

Comment:
```suggestion
Since the MacOS is a UNIX operating system, it is usually easier to set up local programming environment than Windows. As well, it is also among the most supported OS in terms of IT support and app availability. However, MacOS is generally not compatible with non-Apple products. This means that you would need to purchase a MacBook to use MacOS, which may be expensive/less cost-effective depending on your budget and what specifications you're looking for.
```